Common/include/option_structure.hpp

```diff
@@ -1588,6 +1618,8 @@ static const MapType<std::string, ENUM_OBJECTIVE> Objective_Map = {
   MakePair("SURFACE_MOM_DISTORTION", SURFACE_MOM_DISTORTION)
   MakePair("SURFACE_SECOND_OVER_UNIFORM", SURFACE_SECOND_OVER_UNIFORM)
   MakePair("SURFACE_PRESSURE_DROP", SURFACE_PRESSURE_DROP)
+  MakePair("SURFACE_SPECIES_0", SURFACE_SPECIES_0)
+  MakePair("SURFACE_SPECIES_VARIANCE", SURFACE_SPECIES_VARIANCE)
```

Comment:
Something entirely different, historyMap.py
You know, to update shape_optimization in its inoperability

Reply:
I will take a look after lunch how adding CFlowOutput to that list changes things